public/scripts/statistics.js

```diff
@@ -16,9 +16,7 @@ function initializePlayListControl(playListTracks){
       return obj1.activeStat.counter - obj2.activeStat.counter;
     });
     $("#trackList").children().remove();
-    playListTracks.map(function(track){
-      writePlayListToPanel(track);
```

Comment:
these `writePlayListToPanel(track);` calls were what was causing the behavior you contacted me about, since `writePlayListToPanel()` took two arguments and we were only passing it one. javascript--

Reply:
lol did you just decrement javascript? I love it.